pages/api/places/[id].js

```diff
@@ -27,6 +27,23 @@ export default async function handler(request, response) {
 
     response.status(200).json(place);
   }
+  //----------------------------------------------------------------
+
+  if (request.method === "PUT") {
+    try {
+      const { initialReview, description, name, region, activities } =
+        request.body;
+      const updatedPlace = await Place.findByIdAndUpdate(
+        id,
+        { initialReview, description, name, region, activities },
```

Comment:
mongoose should be clever enough to also allow this and only change the fields that need an update:

```suggestion
      const updatedPlaceData =
        request.body;
      const updatedPlace = await Place.findByIdAndUpdate(
        id,
        updatedPlaceData,
```